src/test/java/cloud/fogbow/ras/core/plugins/interoperability/opennebula/compute/v5_4/OpenNebulaComputePluginTest.java

```diff
@@ -97,761 +97,441 @@
-		PowerMockito.mockStatic(VirtualMachine.class);
-		PowerMockito.when(VirtualMachine.allocate(Mockito.any(Client.class), Mockito.anyString())).thenReturn(response);
-		Mockito.when(response.isError()).thenReturn(true);
-		Mockito.when(response.getErrorMessage()).thenReturn(RESPONSE_NOT_ENOUGH_FREE_MEMORY);
-
-		// exercise
-		this.plugin.requestInstance(computeOrder, cloudUser);
-	}
-	
-	// test case: When attempting to allocate a virtual machine with the
-	// requestInstance method call, and an error message occurs with the words limit
-	// and quota, a QuotaExceededException will be thrown.
-	@Test(expected = QuotaExceededException.class) // verify
-	public void testRequestInstanceThrowQuotaExceededException() throws FogbowException {
-		// set up
-		CloudUser cloudUser = createCloudUser();
-		List<String> networkIds = null;
-		ComputeOrder computeOrder = createComputeOrder(networkIds, CPU_VALUE_4, MEMORY_VALUE_2048, DISK_VALUE_8GB);
-		HardwareRequirements flavor = createHardwareRequirements();
-		Client client = Mockito.mock(Client.class);
-		Mockito.doReturn(flavor).when(this.plugin).findSmallestFlavor(client, computeOrder);
+		Mockito.when(virtualMachine.info()).thenReturn(response);
+		Mockito.when(response.isError()).thenReturn(false);
+		Mockito.when(response.getMessage()).thenReturn(xml);
 
-		LaunchCommandGenerator mockLaunchCommandGenerator = Mockito.spy(new DefaultLaunchCommandGenerator());
-		Mockito.doReturn(FAKE_USER_DATA).when(mockLaunchCommandGenerator).createLaunchCommand(Mockito.any());
-		this.plugin.setLaunchCommandGenerator(mockLaunchCommandGenerator);
+		ComputeInstance expected = new ComputeInstance(id, OpenNebulaStateMapper.COMPUTE_RUNNING_STATE, name, cpu, memory, disk, ipAddresses);
 
-		int choice = 1;
-		String networkId = null;
-		String valueOfCpu = String.valueOf(CPU_VALUE_4);
-		String valueOfRam = String.valueOf(MEMORY_VALUE_2048);
-		String valueOfDisk = String.valueOf(DISK_VALUE_8GB);
-		String template = generateTemplate(choice, valueOfCpu, valueOfRam, networkId, valueOfDisk);
+		// exercise
+		ComputeInstance computeInstance = this.plugin.doGetInstance(virtualMachine);
 
-		String message = FIELD_RESPONSE_LIMIT + SEPARATOR + FIELD_RESPONSE_QUOTA;
-		
-		OneResponse response = Mockito.mock(OneResponse.class);
-		PowerMockito.mockStatic(VirtualMachine.class);
-		PowerMockito.when(VirtualMachine.allocate(Mockito.any(Client.class), Mockito.anyString())).thenReturn(response);
-		Mockito.when(response.isError()).thenReturn(true);
-		Mockito.when(response.getErrorMessage()).thenReturn(message);
+		// verify
+		Mockito.verify(virtualMachine, Mockito.times(1)).info();
+		Mockito.verify(virtualMachine, Mockito.times(1)).getId();
+		Mockito.verify(virtualMachine, Mockito.times(1)).getName();
+		Mockito.verify(virtualMachine, Mockito.times(1)).lcmStateStr();
+		Mockito.verify(virtualMachine, Mockito.times(3)).xpath(Mockito.anyString());
+		Mockito.verify(response, Mockito.times(1)).getMessage();
```

Comment:
The same previous suggestion for these verifies.

Reply:
this and other non requestInstance related tests are to be refactored in a follow-up pr. for now, i'm going to fix the issues you mentioned.